lib/data_hygiene/anonymise_email_addresses.sql

```diff
@@ -27,6 +27,10 @@ SET address = CONCAT('anonymous-', a.id, '@example.com')
 FROM addresses a
 WHERE a.address = s.address;
 
+# Deletes all emails that are older than 1 day old.
+DELETE FROM emails
+WHERE created_at < current_timestamp - interval '1 day';
+
```

Comment:
This looks good - you'd want to move this to the start of this process as it does an insert based on all emails that could be a lot simpler with them deleted.

Reply:
Yes, good point! Done :)

Reply:
Also added some tests for good measure

Reply:
Great stuff 👍 

I'm just trying it out (since I've been playing with db syncing) and it might also be prudent to have a db index on created_at for it as it's taking a super long time to run. I'm not really sure how much of a factor in it an index is though. I'll try and test it out.

Reply:
Okay dokay, thanks! 💯